app/controllers/devise_token_auth/passwords_controller.rb

```diff
@@ -181,5 +173,16 @@ def password_resource_params
     def render_not_found_error
       render_error(404, I18n.t('devise_token_auth.passwords.user_not_found', email: @email))
     end
+
+    def validate_redirect_url_param
+      # give redirect value from params priority
+      @redirect_url = params.fetch(
+        :redirect_url,
+        DeviseTokenAuth.default_password_reset_url
+      )
+
+      render_create_error_missing_redirect_url unless @redirect_url
+      render_error_not_allowed_redirect_url if blacklisted_redirect_url?
```

Comment:
To avoid `DoubleRenderError` exception.
```suggestion
      return render_error_not_allowed_redirect_url if blacklisted_redirect_url?
```